modules/devices/x86/processor.c

```diff
@@ -213,66 +214,111 @@ static void __cache_obtain_info(Processor *processor, gint processor_number)
     g_free(endpoint);
 }
 
+static gboolean _g_strv_contains(const gchar * const * strv, const gchar *str) {
```

Comment:
You can use `g_strv_contains()` in the code, and define this function using an `#if !GLIB_CHECK_VERSION(2,44)` block.  If this ends up being used by other parts of the code, it might be a good idea to put outside of a module and somewhere that can be reached by the project as a whole.